packages/next/bin/next.ts

```diff
@@ -129,7 +129,16 @@ if (process.versions.pnp === '3') {
 }
 
 // Make sure commands gracefully respect termination signals (e.g. from Docker)
-process.on('SIGTERM', () => process.exit(0))
+// Allow the graceful termination to be manually configurable
+if (!process.env.NEXT_MANUAL_SIGTERM) {
+  process.on('SIGTERM', () =>
+    setTimeout(
+      () => process.exit(0),
+      parseInt(process.env.NEXT_GRACEFUL_SHUTDOWN_TIMEOUT_MS || '0', 10)
+    )
```

Comment:
Why `setTimeout` though? What about `process.nextTick`?

Reply:
Ah nevermind we would we better off removing the timeout all together, as the user can handle the graceful shutdown however they want it to be.  

Sorry I know it looks like I wanted it to exit out immediately as `0` is passed to the timeout and `process.nextTick` would've prioritized this before than the timeout, but it was more like funny customization.